src/components/routes/Starred.vue

```diff
@@ -21,12 +27,32 @@ export default defineComponent({
 	name: 'RoutesStarred',
 	components: {
 		ContentTemplate,
+		NcCounterBubble,
+	},
+
+	props: {
+		/**
+		 * Unique identifier of the feed whose starred entries this component displays.
+		 *
+		 * @type {number}
+		 * @example 42 or 0
+		 *
+		 * Used to fetch the relevant entries from the backend and to filter the displayed items.
+		 */
+		feedId: {
+			type: Number,
+			required: false,
+			default: 0,
+		},
```

Comment:
```suggestion
	props: {
		/**
		 * Unique identifier of the feed whose starred entries this component displays.
		 *
		 * @type {string}
		 *
		 * Used to fetch the relevant entries from the backend and to filter the displayed items.
		 */
		feedId: {
			type: String,
			required: false,
			default: undefined,
		},
```

The router gives the prop always as String.